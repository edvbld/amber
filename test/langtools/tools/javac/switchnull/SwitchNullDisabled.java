/**
 * @test
 * @compile/fail/ref=SwitchNullDisabled.out -XDrawDiagnostics SwitchNullDisabled.java
 */

public class SwitchNullDisabled {
    private int switchNull(String str) {
        switch (str) {
            case null: return 0;
            case "": return 1;
            default: return 2;
        }
    }
}
