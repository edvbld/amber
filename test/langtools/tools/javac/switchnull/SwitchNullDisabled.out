SwitchNullDisabled.java:10:18: compiler.err.switch.null.not.allowed
1 error
